src/main/scala/akka/contrib/process/NonBlockingProcess.scala

```diff
@@ -242,29 +242,55 @@ class NonBlockingProcess(
     pb.start()
   }
 
+  private var exitCodeToUse: Int = 255
+
   override def receive: Receive = {
+    case s: Started =>
+      context.parent ! s
```

Comment:
Why have we moved this logic - just fire from the handler as before - saves doubling up.